package.json

```diff
@@ -8,7 +8,8 @@
   "scripts": {
     "test": "npm run lint && npm run unit-test",
     "unit-test": "tap -Rspec ./test/lib/*.test.[tj]s --timeout=300",
-    "lint": "tslint -p tsconfig.json",
+    "lint": "eslint --color --cache '{lib,test}/**/*.{js,ts}' && prettier --concurrency=${JOBS:-3} --quiet --check '{lib,test}/**/*.{js,ts}'",
```

Comment:
I bet you would like to use that and also prettify all files in repo 😉

Reply:
![image](https://user-images.githubusercontent.com/40601533/79864882-dfdef580-83e2-11ea-9a8b-ac7ae911bb76.png)